playbook-website/app/javascript/site_styles/_opacity_guidelines.scss

```diff
@@ -0,0 +1,55 @@
+ul.opacity-removing-utilities {
+
+  &.opacity-removing-utilities {
```

Comment:
@cvargaspower this would translate to:

```html
<ul class="opacity-removing-utilities opacity-removing-utilities">
```

I don't know that this was your intent?